packages/ecc-client-ga4gh-tes/package.json

```diff
@@ -1,63 +1,57 @@
-  "type": "module",
-  "version": "0.0.2",
-  "license": "MIT",
-  "repository": "https://github.com/elixir-cloud-aai/cloud-components",
+  "componentsPrefix": "ecc-client-lit-ga4gh-tes-",
   "exports": {
-    ".": "./dist/index.js",
-    "./package.json": "./package.json"
+    ".": { "types": "./dist/index.d.ts", "import": "./dist/index.js" },
+    "./dist/custom-elements.json": "./dist/custom-elements.json",
+    "./dist/index.js": "./dist/index.js",
+    "./dist/components/*": "./dist/components/*",
+    "./dist/react": "./dist/react/index.js",
+    "./dist/react/*": "./dist/react/*"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/elixir-cloud-aai/cloud-components.git"
   },
-  "files": [
-    "images",
-    "dist"
-  ],
-  "keywords": [
-    "TES",
-    "Task Execution Service",
-    "Elixir",
-    "Cloud Component",
-    "Web Component"
-  ],
   "scripts": {
-    "dev": "npm run build && concurrently -k -r \"npm run build -- --watch\" \"wds\"",
-    "lint:fix": "npm run lint -- --fix",
-    "lint": "eslint --ext .ts,.html . --ignore-path .gitignore",
+    "analyze": "cem analyze --litelement",
+    "build": "node ../../scripts/build.js",
+    "dev": "concurrently -r \"npm run build -- --watch\" \"wds\"",
+    "clean": "rm -rf dist node_modules src/react custom-elements-manifest.config.js",
     "test": "wtr --coverage",
     "test:watch": "wtr --watch",
-    "build": "tsup src/index.ts --format cjs,esm --dts",
-    "start:build": "web-dev-server --root-dir dist --app-index index.html --open",
-    "analyze": "cem analyze --fast",
-    "release": "npm run build && npm publish"
-  },
-  "dependencies": {
-    "@microsoft/fast-components": "^2.30.6",
-    "@microsoft/fast-element": "^1.12.0",
-    "@microsoft/fast-foundation": "^2.49.0"
+    "lint": "npx eslint .",
+    "lint:fix": "npm run lint -- --fix",
+    "prepublish": "npm run build"
   },
+  "dependencies": { "@elixir-cloud/design": "*", "lit": "^2.8.0" },
   "devDependencies": {
-    "@babel/preset-env": "^7.16.4",
     "@custom-elements-manifest/analyzer": "^0.4.17",
     "@open-wc/eslint-config": "^9.2.1",
     "@open-wc/testing": "^3.1.6",
-    "@typescript-eslint/eslint-plugin": "^5.59.8",
+    "@typescript-eslint/eslint-plugin": "^5.48.0",
+    "@typescript-eslint/parser": "^5.48.0",
     "@web/dev-server": "^0.1.34",
-    "@web/dev-server-esbuild": "^0.4.3",
     "@web/test-runner": "^0.14.0",
-    "babel-plugin-template-html-minifier": "^4.1.0",
     "concurrently": "^5.3.0",
-    "deepmerge": "^4.2.2",
     "eslint": "^8.31.0",
-    "eslint-config-airbnb-typescript": "^17.0.0",
-    "eslint-plugin-prettier": "^4.2.1",
-    "rimraf": "^3.0.2",
+    "eslint-config-elixir": "*",
+    "eslint-config-prettier": "^8.3.0",
     "tslib": "^2.3.1",
     "tsup": "^7.2.0",
-    "typescript": "^4.9.5"
+    "typescript": "*",
+    "@lit/react": "*",
+    "react": "*",
+    "commander": "*",
+    "custom-element-jet-brains-integration": "*",
```

Comment:
if we are doing clean up lets figure out if these are needed? @SalihuDickson

Reply:
yh they are needed, the react packages are needed, `@lit/react` and `react` are needed for packaging the react components. commander is needed to run the cem config that generates the manifest so that vscode, jetbrains, and some other supported  editors know how to handle the components. And the last package is also needed for generating the jetbrains manifest for custom components. I think there's even one package missing. I'll have to look into it.

Reply:
But the point is do we need manifest files in the first place, if I am not wrong it can be used in very specific part of debugging, I have rarely used that kind of debugging.

Reply:
well we don't **need** the manifest but it's a nice to have. For now the main plus is that it allows devs working with our components to enable autocomplete in regular html and other supported frameworks, and hover over components to get more info, as well as a documentation link to jump to the docs for a specific component, the same way you can jump to an mdn reference for regular components. You can read more about it [here](https://github.com/microsoft/vscode-custom-data) and [here](https://github.com/JetBrains/web-types). 

It also allows the possibility of developing tools that can take advantage of a properly defined web compnents manifest standard. For example we're currently using the regular manifest to  auto generate react components.

Reply:
Agree with @SalihuDickson here, it's best to have them generated for the devs using our library.